### ISSUE TITLE
Tensorflow 1.8 with CUDA 9.2

### ISSUE THREAD

**Issue by XuYunqiu** [fo40225/tensorflow-windows-wheel#12]
Have you compiled Tensorflow 1.8 with CUDA 9.2?

**Comment by fo40225** [comment]
I will use CUDA 9.2 to build tensorflow-gpu 1.9.0.

**Comment by elranu** [comment]
yes and the last cudnn drivers are Download cuDNN v7.1.4 (May 16, 2018), for CUDA 9.2. Please do it with those. Because I cannot find a download link cuDNN v7.1.3

Thanks for your work!

**Comment by fo40225** [comment]
cuDNN 7.1.3 for CUDA 9.1 Windows 10

cudnn-9.1-windows10-x64-v7.1.zip

SHA1: 914006C6407256408EE9C18D6115FDA1209C2F0B

[Google Drive](https://drive.google.com/open?id=1EXikw2Gtgteo3gEw87PrIx7K4Hxv4kXg)

[百度盤](https://pan.baidu.com/s/1vDPmlpt_ABpPF8O6XKcfRA)

**Comment by fo40225** [comment]
I have rebuild the tensorflow-gpu 1.8.0 with CUDA 9.2& cuDNN 7.1.4.

https://github.com/fo40225/tensorflow-windows-wheel/tree/master/1.8.0/py36/GPU/cuda92cudnn71sse2

try it.

**Comment by elranu** [comment]
@fo40225  thanks for the last version of cuda 9.2. It works to me much better that the 9.1. 

Can you compile me the version with avx support? I think it will work even better ;)

Thanks for your work! I really appreciate it :)

**Comment by fo40225** [comment]
Tensorflow team has planed to use CUDA 9.2 to build 1.9.0, it will contain AVX support.

**Comment by LSinev** [comment]
Offtopic: CUDA 9.1 + AVX (not AVX2) to build 1.8.0 maybe?

**Comment by fo40225** [comment]
Nvidia didn't provide cuDNN 7.1.x for CUDA 9.1, maybe cuda 9.1 is deprecated.

**Comment by Lenchik** [comment]
> Nvidia didn't provide cuDNN 7.1.x for CUDA 9.1

[cudnn-9.1-windows10-x64-v7.1.zip](https://drive.google.com/open?id=1EXikw2Gtgteo3gEw87PrIx7K4Hxv4kXg) - this is your link [from above](https://github.com/fo40225/tensorflow-windows-wheel/issues/12#issuecomment-390462502)

**Comment by jacano** [comment]
I think it's kind of hidden, because you can download it from here:
https://developer.nvidia.com/compute/machine-learning/cudnn/secure/v7.1.2/prod/9.1_20180316/cudnn-9.1-windows10-x64-v7.1

**Comment by gaohongfein** [comment]
Have you compiled Tensorflow 1.8 with CUDA 9.2 in Linux for example ubuntu？

**Comment by fo40225** [comment]
@gaohongfein No, you can ask others who maintains custom Linux build for help.

https://github.com/mind/wheels
https://github.com/lakshayg/tensorflow-build
https://github.com/yaroslavvb/tensorflow-community-wheels/issues

**Comment by gaohongfein** [comment]
Thank you very much

**Comment by mtianyan** [comment]
@gaohongfein  I build the cuda9.2 for TensorFlow1.8.0(python3.6)on ubuntu18.04。
https://github.com/mtianyan/tensorflow-linux-wheel

**Comment by sleeplessai** [comment]
Ask something about cuda92cudnn71sse2 wheel. Is there AVX2 support on it? 
// And thanks a lot for compiling these packages, later i would try c++ lib.

**Comment by fo40225** [comment]
@yanhaiming56 You have two options to solve that:

- edit `C:\Program Files\NVIDIA GPU Computing Toolkit\CUDA\v9.2\include\crt\host_config.h` Line 131:

from 
``` cpp
#if _MSC_VER < 1600 || _MSC_VER > 1913
```
to
``` cpp
#if _MSC_VER < 1600
```

### OR

-  install vs2017 15.6 from https://docs.microsoft.com/en-us/visualstudio/productinfo/installing-an-earlier-release-of-vs2017

**Comment by tgbamg** [comment]
Is this wheel(Tensorflow 1.8 with CUDA 9.2)support python2.7?
And where should I find a CUDA9.2 wheel which support python2.7?

**Comment by fo40225** [comment]
@tgbamg Uploaded. 

https://github.com/fo40225/tensorflow-windows-wheel/tree/master/1.8.0/py27/GPU/cuda92cudnn71sse2

**Comment by jlar0che** [comment]
I want to second @elranu and give massive props to @fo40225 for compiling the whl file with Cuda 9.2, CuDNN 7.14 and Python 3.6 support in Windows. [Your whl file](https://github.com/fo40225/tensorflow-windows-wheel/tree/master/1.8.0/py36/GPU/cuda92cudnn71sse2) worked like a charm (see my installation environment and hardware details below).

Would you be able to compile a version with AVX support? If so, you would quickly earn yourself the title of "God Status" ;)

Thanks so much for your work - we all appreciate it!

**My specs that worked with the WHL:**

- Windows 10 64bit (all latest Windows updates installed)
- Python 3.6.3 (64bit) [NOTE: I'm just using simple virtual environments with virtualenv v15.1.0 & virtualenvwrapper-win v1.2.3 - NOT Anaconda]
- Visual Studio Community 2017
- CUDA Toolkit 9.2.148 (for Windows 10)
- CuDNN 7.1.4 (for Cuda 9.2)
- fo40225's compiled WHL: tensorflow_gpu-1.8.0-cp36-cp36m-win_amd64.whl

**Comment by RobMcH** [comment]
@fo40225 Will you build tensorflow-gpu with CUDA 10 support?

**Comment by fo40225** [comment]
I will use CUDA 10 to build tensorflow-gpu 1.11 for python 3.6 x64.

**Comment by bluecd** [comment]
> I will use CUDA 10 to build tensorflow-gpu 1.11 for python 3.6 x64.

Did you succeeded w/above?

**Comment by fo40225** [comment]
> > I will use CUDA 10 to build tensorflow-gpu 1.11 for python 3.6 x64.
> 
> Did you succeed w/above?

@bluecd Yes.

https://github.com/fo40225/tensorflow-windows-wheel/tree/master/1.11.0/py36/GPU
